src/apollos/models/likes/model.ts

```diff
@@ -0,0 +1,76 @@
+import { Cache, defaultCache } from "../../../util/cache";
+import { MongoConnector } from "../../mongo";
+import { parseGlobalId, createGlobalId } from "../../../util/node/model";
+
+export interface LikeDocument {
+  _id: string;
+  userId: string;
+  entryId: string;
+  type: string;
+  createdAt: Date;
+}
+
+const schema: Object = {
+  userId: String,
+  entryId: String,
+  type: String,
+  createdAt: String,
+};
+
+const Model = new MongoConnector("like", schema);
+
+export class Like {
+  public __type: string = "Like";
+  public model: MongoConnector;
+  public cache: Cache;
+
+  constructor({ cache }: { cache: Cache | any } = { cache: defaultCache }) {
+    this.model = Model;
+    this.cache = cache;
+  }
+
+  public async getFromUserId(userId: string): Promise<LikeDocument[]> {
+    return await this.cache.get(`${this.__type}:${userId}`, () => (
+      this.model.find({ userId })
+    )) as LikeDocument[];
+  }
+
+  public async getLikedContent(userId: string, contentModel: any): Promise<any[]> {
+    const likes = await this.getFromUserId(userId);
+    return await likes.map(async (like) => {
```

Comment:
@jbaxleyiii i'm not sure either. it seems to be working though when testing with multiple likes. i'm open to changing it though